java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -379,6 +379,29 @@ public byte getByte(long index) {
     return offHeap.data.getByte(index * type.sizeInBytes);
   }
 
+  /**
+   * Get the byte list at index.
+   */
+  public byte[] getByteList(long index) {
+    assert type == DType.LIST;
```

Comment:
Sadly we need to also verify that the child type is `DType.INT8`, which I think will require a JNI call, so we need to be careful that we fully encase it in the assert so it is not called when assertions are off. We also need to verify that the child has no nulls in it or we will be returning bogus data.  Again this should be in the assert code.  This is because DType.STRING implies that the child column can not have any nulls in it, so we pull the child data up into the ColumnVector, but DType.LIST has no such guarantee so we keep it in the child so it can have a validity mask.  We may update DType.STRING in the future to look like a LIST for consistency.